ui/pages/first-time-flow/metametrics-opt-in/index.scss

```diff
@@ -33,10 +33,6 @@
 
   &__body-graphic {
     margin-top: 25px;
-
-    .fa-bar-chart {
-      color: var(--color-icon-default);
-    }
   }
 
   &__description {
```

Comment:
Not used

![Screen Shot 2022-05-16 at 5 31 15 PM](https://user-images.githubusercontent.com/8112138/168915478-b4948f38-fb6e-496d-8814-4ee06cd054f4.png)